crypto/zkp/primitives/qrspecialrsaproofs/representation.go

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2017 XLAB d.o.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package qrspecialrsaproofs
+
+import (
+	"math/big"
+
+	"github.com/xlab-si/emmy/crypto/common"
+	"github.com/xlab-si/emmy/crypto/groups"
+)
+
+// RepresentationProver is like SchnorrProver but in a QRSpecialRSA group (note that here proof data is
+// is computed in Z, not modulo as in Schnorr). Also, RepresentationProver with only one base and one secret
+// is very similar to the DFCommitmentOpeningProver (RepresentationProver does not have a committer though).
+type RepresentationProver struct {
+	group      *groups.QRSpecialRSA
+	l_r        int // security parameter
```

Comment:
Could you please rename this variable to either something more descriptive (for instance, `secParam`), or maybe `k` that is commonly used to denote security parameter - something that does not require underscores? The convention is to use mixedCaps.

This comment applies to all usages of `l_r`.